Directory.Build.targets

```diff
@@ -1,8 +1,8 @@
 <Project>
   <Sdk Name="Microsoft.Build.CentralPackageVersions" Version="2.0.52" />
   <!-- Need full debugtype for code coverage -->
-	<PropertyGroup Condition="'$(Configuration)' == 'Debug'">
-		<DebugType>portable</DebugType>
+	<PropertyGroup Condition="'$(Configuration)' == 'Debug' and ('$(BuildingInsideVisualStudio)' != '' or '$(CodeCoverageBuild)' == 'true')">
```

Comment:
Don't you still need to define the case for it being portable? Or does it default to that already if not specified?

Reply:
portable is the default. It's assigned in directory.build.props, too, just to be sure.